rojo-test/syncback-tests/ignore_paths_removing/expected/default.project.json

```diff
@@ -4,18 +4,6 @@
     "$className": "DataModel",
     "Workspace": {
       "$properties": {
-        "CSGAsyncDynamicCollision": {
-          "Enum": 0
-        },
-        "DecreaseMinimumPartDensityMode": {
-          "Enum": 0
-        },
-        "MoverConstraintRootBehavior": {
-          "Enum": 0
-        },
-        "RenderingCacheOptimizations": {
-          "Enum": 0
-        },
```

Comment:
Is this the behavior we want (remove default properties even if they are explicitly specified in the json files)?

Users might not expect this to happen... but if someone uses an out-of-date reflection db for syncback, then later upgrades, and the implementation does not remove default props when explicitly specified, then they will be left with redundant properties